src/access/accesscontrol/accesscontrol.cairo

```diff
@@ -91,7 +91,7 @@ mod AccessControlComponent {
         ///
         /// Requirements:
         ///
-        /// - The caller must have `role`'s admin role.
+        /// - The caller must have ``role``'s admin role.
```

Comment:
The highlighting with backticks can get weird when it ends with an apostrophe instead of a space...maybe it's an asciidoc thing because I can't reproduce the effect in MD. Will update

Reply:
but this in-code docs aren't asciidocs are they?